### DISCUSSION TITLE
Can this plugin be used with jQuery?

### DISCUSSION THREAD

**Discussion by terry2010** [zhensherlock/watermark-js-plus#362] [Q&A] [answered]
We have many projects in the company that use jQuery and need to add a watermark feature. After testing, we found that this plugin works very well. Could anyone please provide instructions on how to use this plugin in pages using jQuery?

Best regards.

**Comment by zhensherlock** [comment, marked as answer]
https://stackblitz.com/edit/js-y7xfw5?file=index.html
This is an example using jQuery.

**Reply by terry2010** [reply]
Thank you for your great response. May God bless you.
